docs/getting-started/quickstart/README.md

```diff
@@ -383,42 +385,40 @@ code ./sample_append.js
 Alternatively, you can locate and open the file from the EXPLORER window on the left of Codespaces. You can find the path to the file in the command above.
 :::
 
-2. In step 1 of the code, review how the client connects to EventStoreDB:
+2. In step 1 of the code, review how the client connects to KurrentDB:
 
 ::: tabs#dev-language-platform
 @tab Select >
 
 @tab Python
 ```python
-# Create an instance of EventStoreDBClient, connecting to the EventStoreDB at localhost without TLS
-client = EventStoreDBClient(uri="esdb://localhost:2113?tls=false")  
+# Create an instance of KurrentDBClient, connecting to the KurrentDB at localhost without TLS
+client = KurrentDBClient(uri="kurrentdb://localhost:2113?tls=false")  
 ```
 
 @tab Java
 ```java
-// configure the settings to connect to EventStoreDB locally without TLS
-EventStoreDBClientSettings settings = EventStoreDBConnectionString.
-   parseOrThrow("esdb://localhost:2113?tls=false");
-
+// configure the settings to connect to KurrentDB locally without TLS
+KurrentDBClientSettings settings = KurrentDBConnectionString.parseOrThrow("kurrentdb://localhost:2113?tls=false");
 // apply the settings and create an instance of the client
-EventStoreDBClient client = EventStoreDBClient.create(settings); 
+KurrentDBClient client = KurrentDBClient.create(settings);
 ```
 
 @tab .NET
 ```c#
-// Create an instance of EventStoreDBClient, connecting to the EventStoreDB at localhost without TLS
-var settings = EventStoreClientSettings.Create("esdb://localhost:2113?tls=false");
+// Create an instance of EventStoreClientSettings, connecting to the KurrentDB at localhost without TLS
+var settings = EventStoreClientSettings.Create("kurrentdb://localhost:2113?tls=false");
```

Comment:
Just to confirm - is `EventStoreClientSettings` intentional here (and further down in this file?) I've not used the rebranded clients yet, so I don't know if this has changed or not

Reply:
Yes! .net client haven't been rebranded yet. This will come later